localization/english/dvg_names_l_english.yml

```diff
@@ -74,3 +74,144 @@
  Suleiman:0 "Suleiman"
  Phillip:0 "Phillip"
  Victor_Emmanuel:0 "Victor Emmanuel"
+
+#Quechua Names
+
+#Male First Names
+Atawallpa: "Atawallpa"
+Wayna: "Wayna"
+Wanka: "Wanka"
+Waskhar: "Waskhar"
+Rimaq: "Rimaq"
+Ruminaki: "Ruminaki"
+Sapay: "Sapay"
+Sumainka: "Sumainka"
+Tupakusi: "Tupakusi"
+Ukumari: "Ukumari"
+Wallpaya: "Wallpaya"
+Wamay: "Wamay"
+Tupac: "Túpac"
+Anti: "Anti"
+Anqasmayu: "Anqasmayu"
+Chikan: "Chikan"
+Inti: "Inti"
```

Comment:
This vould conflict, but it shouldn't

Reply:
I can help you work through any issues that arise when we get around to merging that commit.